docs/src/man/getting_started.md

```diff
@@ -571,7 +576,7 @@ a function object that tests whether each value belongs to the subset
     - when `view` or `@view` is used (e.g. `@view df[1:3, :A]`).
 
     More details on copies, views, and references can be found
-    [here.](https://juliadata.github.io/DataFrames.jl/stable/lib/indexing/#getindex-and-view-1)
+    in [`getindex` and `view`](@ref).
```

Comment:
@mortenpi - In this case the header after `###` contains backquotes. Again - the question is if referencing it like this is OK.

Reply:
Yep, should be.

Reply:
Yep, should be.